test/test_nn.py

```diff
@@ -10116,6 +10116,59 @@ def test_grid_sample_3d_alert_nondeterministic(self, device):
         output = F.grid_sample(input, grid, align_corners=False)
         output.sum().backward()
 
+    @dtypes(torch.float, torch.double)
+    @largeTensorTest(lambda self, device, dtype:
+                     32769 * (65536 + 3 * 65536 / 128) *
+                     torch.tensor([], dtype=dtype).element_size())
```

Comment:
I don't think the double test case will ever run on a GPU (32769 * (65536 + 3 * 65536 / 128) * 8 bytes is 17.5 gb). Could we reduce the size of the test case to use an image of size `[1, 1, 32769, 32769]` or something smaller instead?

Reply:
Resolved; we need this much memory to test the algorithm